config/initializers/breakers.rb

```diff
@@ -115,4 +115,8 @@
   Breakers.redis_prefix = ''
   Breakers.client = client
   Breakers.disabled = true if Settings.breakers_disabled
+
+  Breakers.config&.outage_exceptions += [
+    Common::Exceptions::BackendServiceException
```

Comment:
Where does `Breakers.config` come from? I don't see it used anywhere else in vets-api 😕